browser/ntp_background_images/ntp_custom_background_images_service_delegate.cc

```diff
@@ -28,7 +29,7 @@ bool NTPCustomBackgroundImagesServiceDelegate::IsCustomBackgroundEnabled() {
   if (prefs->IsManagedPreference(prefs::kNtpCustomBackgroundDict))
     return false;
 
-  return prefs->GetBoolean(kNewTabPageCustomBackgroundEnabled);
+  return NTPBackgroundPref(profile_->GetPrefs()).IsCustomImageType();
```

Comment:
also this class should have consistent naming re NTPBackgroundImage and should not be in a namspace

Reply:
Removed namespace